src/System.Device.Ports/System.Device.Ports.SerialPort/System.Device.Ports.SerialPort.csproj

```diff
@@ -0,0 +1,57 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(NETCoreSdkRuntimeIdentifier.StartsWith('win'))">$(TargetFrameworks);net6.0-windows10.0.17763.0</TargetFrameworks>
```

Comment:
Let's also for now remove the windows-specific build and try to for now just work on a RID-less configuration. We can later add a windows vs linux specific build later when we need to.